Creating a new file throws "PGN offsets not found for file" from the Files tab

### Describe the bug

Creating a new file from the Files tab always throws "PGN offsets not found for file" error notification
Importing a game from the Board Tab - Import Game doesn't throw the same error.

### Reproduction

1. Open app and navigate to the Files tab
2. Click Create
3. Input a name and PGN game (or leave PGN game empty, it's also reproable that way)
4. Click Create. Notice the error notification poping up


### Platform and versions

Chessifier
Version: 0.3.2

Tauri version: 2.7.0

OS: windows x86_64 10.0.26100

### Stack trace

```Shell
[2025-08-16][21:19:10][webview:An@http://tauri.localhost/assets/index.js:657:223][ERROR] PGN offsets not found for file
```

--- a/src/utils/files.ts
+++ b/src/utils/files.ts
@@ -71,11 +71,14 @@ export async function createFile({
   };
   await writeTextFile(file, pgn || makePgn(defaultGame()));
   await writeTextFile(file.replace(".pgn", ".info"), JSON.stringify(metadata));
+  
+  const numGames = unwrap(await commands.countPgnGames(file));
+  
   return Result.ok({
     type: "file",
     name: filename,
     path: file,
-    numGames: 1,
+    numGames,
     metadata,
     lastModified: new Date().getUTCSeconds(),
   });

